docs/MAINTAINERS.md

```diff
@@ -0,0 +1,261 @@
+<td><b>Groups:</b> <br /><br /><ul>
+    <li><a href="https://github.com/orgs/intelsdi-x/teams/plugin-maintainers">@intelsdi-x/plugin-maintainers</a></li>
+</ul></td></tr>
+<tr><td colspan="3"></td></tr>
+<tr><td colspan="3"><h4><a name="kjlyon" href="#kjlyon">#</a> Kelly Lyon</h4></td></tr>
+<td><img src="https://avatars.githubusercontent.com/u/21182867?v=3" width="90px" /></td>
+<td><b>Github:</b> <a href="https://github.com/kjlyon">@kjlyon</a><br /></td>
+<td><b>Groups:</b> <br /><br /><ul>
+    <li><a href="https://github.com/orgs/intelsdi-x/teams/snap-maintainers">@intelsdi-x/snap-maintainers</a></li>
+</ul></td></tr>
+<tr><td colspan="3"></td></tr>
+<tr><td colspan="3"><h4><a name="mkleina" href="#mkleina">#</a> Mateusz Kleina</h4></td></tr>
+<td><img src="https://avatars.githubusercontent.com/u/18285077?v=3" width="90px" /></td>
+<td><b>Github:</b> <a href="https://github.com/mkleina">@mkleina</a><br /><br /><b>Company:</b> @intelsdi-x </td>
+<td><b>Groups:</b> <br /><br /><ul>
+    <li><a href="https://github.com/orgs/intelsdi-x/teams/plugin-maintainers">@intelsdi-x/plugin-maintainers</a></li>
+</ul></td></tr>
+<tr><td colspan="3"></td></tr>
+<tr><td colspan="3"><h4><a name="lmroz" href="#lmroz">#</a> Łukasz Mróz</h4></td></tr>
+<td><img src="https://avatars.githubusercontent.com/u/13198817?v=3" width="90px" /></td>
+<td><b>Github:</b> <a href="https://github.com/lmroz">@lmroz</a><br /><br /><b>Mail:</b> <a href="lukasz.mroz@intel.com">lukasz.mroz@intel.com</a><br /><b>Company:</b> Intel Corporation</td>
+<td><b>Groups:</b> <br /><br /><ul>
+    <li><a href="https://github.com/orgs/intelsdi-x/teams/plugin-maintainers">@intelsdi-x/plugin-maintainers</a></li>
+</ul></td></tr>
+<tr><td colspan="3"></td></tr>
+<tr><td colspan="3"><h4><a name="mkuculyma" href="#mkuculyma">#</a> Maria Kucułyma</h4></td></tr>
+<td><img src="https://avatars.githubusercontent.com/u/6523391?v=3" width="90px" /></td>
+<td><b>Github:</b> <a href="https://github.com/mkuculyma">@mkuculyma</a><br /><br /><b>Mail:</b> <a href="maria.kuculyma@intel.com">maria.kuculyma@intel.com</a></td>
+<td><b>Groups:</b> <br /><br /><ul>
+    <li><a href="https://github.com/orgs/intelsdi-x/teams/plugin-maintainers">@intelsdi-x/plugin-maintainers</a></li>
+</ul></td></tr>
+<tr><td colspan="3"></td></tr>
+<tr><td colspan="3"><h4><a name="katarzyna-z" href="#katarzyna-z">#</a> Katarzyna Kujawa</h4></td></tr>
+<td><img src="https://avatars.githubusercontent.com/u/16666942?v=3" width="90px" /></td>
+<td><b>Github:</b> <a href="https://github.com/katarzyna-z">@katarzyna-z</a><br /><br /><b>Company:</b> Intel Corporation</td>
+<td><b>Groups:</b> <br /><br /><ul>
+    <li><a href="https://github.com/orgs/intelsdi-x/teams/plugin-maintainers">@intelsdi-x/plugin-maintainers</a></li>
+</ul></td></tr>
+<tr><td colspan="3"></td></tr>
+<tr><td colspan="3"><h4><a name="marea-cobb" href="#marea-cobb">#</a> Marea Cobb</h4></td></tr>
```

Comment:
Now some members can be skipped https://github.com/lmroz/org2docmd/blob/master/maintainersgen/consts.go#L68 [WIP version]